### ISSUE TITLE
Endless splitting

### ISSUE THREAD

**Issue by ghost** [josephsl/soundSplitter#1]
I started using this addon 2 days back, and found it very nice! But I started noticing that even after closing NVDA the audio of the each app, except NVDA still splitted and I nedd to turn on and off the splitter to audio get back for both channels, for each app eg. Whatsapp, Chrome and on.... I noticed it was an bug until version 22.2 but Im using the version 22.03.01.
NVDA version: 2021.3.5
Windows: 21H2 (SO compilation 22000.556)
©

**Comment by josephsl** [comment]
Hi, thanks for raising this issue. I’m no longer maintaining this add-on, so it is highly unlikely that this issue will be resolved at this time (sorry). Thanks.

**Comment by FalkoBabbage** [comment]
I cannot reproduce this error. Youtube automatically goes back to two channels when I close NVDA with NVDA+Q. I am using NVDA 2021.3.4 and addon version 22.03.01
